x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/index/IndexResolution.java

```diff
@@ -79,6 +93,13 @@ public Map<String, FieldCapabilitiesFailure> getUnavailableClusters() {
         return unavailableClusters;
     }
 
+    /**
+     * @return all indices found by field-caps (regardless of whether they had any mappings)
+     */
+    public Set<String> getResolvedIndices() {
```

Comment:
Nit: since there's no setter, you can drop the get -> resolvedIndices()

Reply:
OK. Fixed in next push. I also changed `getUnavailableClusters()` to `unavailableClusters()`.